python/src/ecole/_combinatorial_auction_generator.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+
+from pyscipopt import Model
+
+import ecole.scip
+
+
+class CombinatorialAuctionGenerator:
+    def __init__(self, parameter_generator=None):
+        self.parameter_generator = parameter_generator
+        self.rng = np.random.RandomState()
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        param_dict = next(self.parameter_generator)
+        model = self.generate_instance(**param_dict)
+
+        return ecole.scip.Model.from_pyscipopt(model)
+
+    def seed(self, seed):
+        self.rng.seed(seed)
+
+    def generate_instance(
+        self,
+        n_items=100,
+        n_bids=500,
+        min_value=1,
+        max_value=100,
+        value_deviation=0.5,
+        add_item_prob=0.9,
+        max_n_sub_bids=5,
+        additivity=0.2,
+        budget_factor=1.5,
+        resale_factor=0.5,
+        integers=False,
+        warnings=False,
+    ):
+        """
+		Generate a Combinatorial Auction instance with specified characteristics, and writes

```

Comment:
The doc is tab indented, please change to spaces. Missing [short summary](https://numpydoc.readthedocs.io/en/latest/format.html#id8)

Reply:
Check https://github.com/ds4dm/ecole/blob/2d791d0a3b116284e26ae4d3473bf8dde3cd8ada/python/src/ecole/environment.py#L63 for an example